Resolve 5# Created static homepage

abalanag opened the pull request:
- Created static homepage
- Added a carousel on the first page

abalanag committed:
Resolve 4# Created static homepage

- Created static homepage
- Added a carousel on the first page

danutchindris commented on src/main/resources/footerScripts.html:
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
```
You should place these fragments in `common.html`.

abalanag replied:
fragments added to common.html

danutchindris commented on src/main/resources/static/homepage.html:
```diff
@@ -0,0 +1,74 @@
+<!doctype html>
```
Not sure what you want to do with this, but the `homepage.html` Thymeleaf template should reside in `resources/templates`, not in the `static` folder.
That's why you won't need the `spring.resources.static-locations=classpath:/static/` property in your `application.properties` file.

abalanag replied:
deleted `spring.resources.static-locations=classpath:/static/` from `application.properties` and created controller in order to access homepage

danutchindris commented on src/main/resources/static/homepage.html:
```diff
@@ -0,0 +1,74 @@
+<br><br><br><br><br>
+<center>
+    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+        <style>
```
I think it's bad practice to pollute the markup with CSS code when you can place it in a dedicated file.

abalanag replied:
moved CSS code into a CSS file

danutchindris commented on src/main/resources/static/homepage.html:
```diff
@@ -0,0 +1,74 @@
+<link href="#" th:replace="footerScripts2 :: bootstrap-js"/>
+<link href="#" th:replace="footerScripts1 :: bootstrap-js"/>
+</body>
+</html>
```
Empty line here please.

abalanag replied:
Empty line added

danutchindris reviewed (changes requested):
Please see the review comments and perform the necessary changes.
Thanks!

abalanag committed:
#5 Updated project

Updated the project acording the requests from the pull request comments
Created class HomepageCtroller

abalanag committed:
Added footer and header on homepage

Added the footer and the header for the homepage for the time that will merge all 3 branches.

danutchindris reviewed (approved):
OK
